components/notification/index.tsx

```diff
@@ -221,7 +223,7 @@ function getRCNoticeProps(args: ArgsProps, prefixCls: string) {
 
   return {
     content: (
-      <div className={iconNode ? `${prefixCls}-with-icon` : ''} role="alert">
+      <div id="notification" data-test-id={dataTestId} className={iconNode ? `${prefixCls}-with-icon` : ''} role="alert">
```

Comment:
`dataTestId` is too specific. Please consider provides another prop instead. Or just?
```
notification.open({
      message: <div data-test-id="test">Some notification</div>,
    });
```